include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +112,550 @@ class HighPressureGasTransport : public MultiTransport
+     * calculation from equation 9-4.19.
+     *
+     * @f[
+     *    F_{Q}^{\text{o}} = 1.22 Q^{0.15} \left( 1 + 0.00385 \left ( \left ( T_r - 12 \right ) ^2 \right ) ^{\frac{1}{MW}} sign(T_r - 12 \right ) \right )
+     * @f]
+     *
+     * @param Q  Species-specific constant
+     * @param Tr  Reduced temperature
+     * @param MW  Molecular weight
+     * @return double
+     */
+    double FQ_i(double Q, double Tr, double MW);
+
+    /**
+     * @brief  Returns the polarity correction term for a species based on reduced
+     * temperature, reduced dipole moment, and critical compressibility. Used in
+     * the viscosity calculation.
+     *
+     * Calculates polarity correction term of the Lucas method for a species based
+     * on the reduced temperature(Tr) and molecular weight(MW). Equation 9.4.18.
+     *
+     * @f[
+     *  \begin{equation}
+     *   F_P^0 =
+     *   \begin{cases}
+     *       1 & 0 \leq \mu_r < 0.022 \\
+     *      1 + 30.55(0.292 - Z_c)^{1.72} & 0.022 \leq \mu_r < 0.075 \\
+     *      1 + 30.55(0.292 - Z_c)^{1.72} \times 0.96 + 0.1(T_r - 0.7) & 0.075 \leq \mu_r
+     *   \end{cases}
+     *  \end{equation}
+     *
+
+     * @note The original description in Poling(2001) neglects to mention what happens
+     * when the quantity raised to the 1.72 power goes negative. That is an undefined
+     * operation that generates real+imaginary numbers. For now, we
+     * take the absolute value of the argument.
+     *
+     * @param mu_r  Species Reduced dipole moment
+     * @param Tr  Reduced temperature
+     * @param Z_c  Species Critical compressibility
+     * @return double
+     */
+    double FP_i(double mu_r, double Tr, double Z_c);
+
+};
+
+
+//These are the parameters that are needed to calculate the viscosity using the Chung method.
+struct ChungMixtureParameters
+{
+    // Mixture critical properties used by the Chung viscosity model.
+    double Vc_mix = 0;
+    double Tc_mix = 0;
+
+    // Values associated with the calculation of sigma and the molecular weight used in the Chung viscosity model.
+    double sigma_mix = 0;
+    double epsilon_over_k_mix = 0;
+    double MW_mix = 0;
+
+    // Values associated with the calculation of the Fc factor in the Chung viscosity model.
+    double mu_mix = 0;
+    double mu_r_mix = 0;
+    double acentric_factor_mix = 0;
+    double kappa_mix = 0;
+};
+
+//! Class ChungHighPressureTransport implements transport properties for
+//! high pressure gas mixtures.
+/*!
+ * The implementation employs a method of corresponding states, using the Takahashi
+ * @cite takahashi1975 approach for binary diffusion coefficients (using mixture
+ * averaging rules for the mixture properties), and the Chung method for the viscosity
+ * and thermal conductivity of a high-pressure gas mixture. All methods are described
+ * in Poling et al. @cite poling2001 (viscosity in Ch. 9, thermal conductivity in
```

Comment:
They both utilize the takahashi function, which is defined externally to both of the classes. I don't want to have the Chung class inherit from the HighPressure class currently because they are quite different and I don't think that a base/derived relationship properly captures the relationship between the two classes.